addons/website/models/website.py

```diff
@@ -296,10 +296,10 @@ def get_cta_data(self, website_purpose, website_type):
 
     @api.model
     def get_theme_snippet_lists(self, theme_name):
-        default_snippet_lists = http.addons_manifest['theme_default'].get('snippet_lists', {})
-        theme_snippet_lists = http.addons_manifest[theme_name].get('snippet_lists', {})
-        snippet_lists = {**default_snippet_lists, **theme_snippet_lists}
-        return snippet_lists
+        return {
+            **load_manifest('theme_default').get('snippet_lists', {}),
+            **load_manifest(theme_name).get('snippet_lists', {})
```

Comment:
Same remark as above, the key `snippet_lists` is always present:
```suggestion
            **load_manifest('theme_default')['snippet_lists'],
            **load_manifest(theme_name)['snippet_lists'],
```